integrations-and-supported-tools/pytorch-lightning/notebooks/Neptune_PyTorch_Lightning.ipynb

```diff
@@ -164,7 +164,9 @@
    "source": [
     "## Define Lightning model\n",
     "\n",
-    "Define a Lightning model with both inbuilt logging (`self.log()`) and custom logging (`neptune_logger.experiment`) to log metrics to Neptune."
+    "Define a Lightning model with both inbuilt logging (`self.log()`) and custom logging (`neptune_logger.experiment`) to log metrics to Neptune.\n",
+    "\n",
+    "`neptune_logger.experiment` is a reference to the Neptune run object created by the `NeptuneLogger()` instance. It can be used to log outside the `prefix` passed to `NeptuneLogger()`, and accepts Neptune's logging methods like `append()` and `upload()`."
```

Comment:
```suggestion
    "`neptune_logger.experiment` is a reference to the Neptune run object created by the `NeptuneLogger()` instance. You can use it to log outside the `prefix` passed to `NeptuneLogger()`. It accepts the Neptune's logging methods such as `append()` and `upload()`."
```

Reply:
@szaganek - can you update the suggestion so that I can add it to the batch?
"It accepts the Neptune's logging methods..." > "It accepts Neptune's logging methods..."